_posts/2021-06-09-running-a-full-node.markdown

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: 'Running a Full Cardano Node'
+categories:
+- 'Getting Started'
+last_modified_at: 2021-06-09
+---
+
+As a developer the best starting point to Cardano is to get a good
+understanding of the Cardano node. This knowledge is essential whether
+you want to create Dapps, wallets, integration tools, mint custom tokens
+or operate your own stake pool.
+
+We will break it down in five basic steps:
+
+1. Set up your Linux environment
```

Comment:
Not super important but is it possible to add a custom class attribute to this ordered list like https://learn.lovelace.academy/getting-started/running-a-full-node/ ? `<ol class="ordered-list">`

Reply:
I added it but the way it works is added to every <ol> if you want to be selective you'd need to use HTML

Reply:
Custom HTML is good for now